packages/maskbook/src/utils/shadow-root/usePortalShadowRoot.tsx

```diff
@@ -64,7 +64,7 @@ function useEffectRef<T>(f: () => T): T {
 }
 
 export function createShadowRootForwardedComponent<
-    T extends { container?: Element | (() => Element | null) | null | undefined }
+    T extends { container?: Element | (() => Element | null) | null | undefined; open: boolean }
```

Comment:
Property 'open' is missing in type 'PropsWithChildren<{ container?: Element | (() => Element | null) | null | undefined; }>' but required in type 'MenuProps'